src/main/java/no/ndla/taxonomy/util/HashUtil.java

```diff
@@ -28,6 +28,10 @@ public static String mediumHash(Object original) {
         return generateHash(original, 8);
     }
 
+    public static String semiHash(Object original) {
+        return generateHash(original, 12);
```

Comment:
Oi denne har ikke jeg sett før. Er ikke denne ganske utsatt for å støte på kollisjoner? Spesielt de korte variantene?

Reply:
Jo dei korte er nok det. Men den brukes for versjoner av taksonomi, og vil ikkje bli veldig mange. 
Eg testa lokalt med lengde 8 og fikk ingen kollisjoner på ca 33000 noder, så tenker lengde 12 bør være innafor.